# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../docpp/flex-hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
The white space between ']' and ']' is necessary or else flex will discard
all closing braces and brackets of the corresponding line in docify.cc.
Patch required with flex version 2.5.31, not required with 2.5.4a.

--- doc++-3.4.10/src/docify.ll	2001-03-08 08:36:20.000000000 +0000
+++ doc++-3.4.10-patched/src/docify.ll	2004-01-13 22:12:27.000000000 +0000
@@ -314,7 +314,7 @@
 	{
 	if(argv[1][0] == '-')
 	    {
-	    fprintf(stderr, "Usage: docify [infile [outfile]]\n");
+	    fprintf(stderr, "Usage: docify [ infile [ outfile ] ]\n");
 	    return 0;
 	    }
 	ain = fopen(argv[1], "r");
